unit2/README.md

```diff
@@ -253,12 +254,16 @@ Opposite situation will be if we'll try to append elements to slice concurrently
 3. any reads or writes of the variable it is locking
 4. any writes to the variable it is locking
 
+#### Answer: 3
```

Comment:
Mutex itself is not aware of memory you're trying to lock. Mutex just locks  other concurrent calls of Mutex.Lock()